jetty-webapp/src/main/java/org/eclipse/jetty/webapp/StandardDescriptorProcessor.java

```diff
@@ -97,7 +96,6 @@ public StandardDescriptorProcessor()
             registerVisitor("filter", this.getClass().getMethod("visitFilter", __signature));
             registerVisitor("filter-mapping", this.getClass().getMethod("visitFilterMapping", __signature));
             registerVisitor("listener", this.getClass().getMethod("visitListener", __signature));
-            registerVisitor("distributable", this.getClass().getMethod("visitDistributable", __signature));
```

Comment:
The `distributable` element has been removed in Servlet 4.0?
Even though it's been removed, I would leave the "visit" and eventually write a comment in `visitDistributable` why we do nothing.
Removing it from here seems like a mistake or an overlook ("hey, they forgot to visit `distributable`").

Reply:
No, it hasn't been removed. The handling of it has just been refactored to WebDescriptor, where it should have been from the beginning, along with other "metadata" about the descriptor (eg is metadata-complete, version etc), not artifacts like servlets/filters etc that is handled by StandardDescriptorProcessor.

Reply:
@janbartel yes I saw that handling of `distributable` has been moved to `WebDescriptor`.
However, I would still keep the `registerVisitor("distributable", ...)` here because it's the one place where I can verify that all elements of a descriptor are actually visited - it's just for completeness so that when someone else comes here it won't say "oh, we don't visit `distributable`" and has to reverse engineer where is it actually handled. The empty `visitDistributable()` method can have a comment that says where it is handled.

Reply:
@janbartel in other words, why is `distributable` different from the others so that it must be handled differently?
The original code of calling `((WebDescriptor)descriptor).setDistributable(true)` made sense.

Reply:
"Distributable" is just like the handling of the "version" of the webapp, or the "metadata-complete" attribute of the webapp: they are meta-data about the descriptor/webapp.  We've never had a visitWebApp method on StandardDescriptorProcessor for example, so it's not and never has been the case that StandardDescriptorProcessor has visited everything in the webapp - some things you have to visit first, before you know what you can visit!

Moreover, I don't find it correct that the StandardDescriptorProcessor calls setters on the Descriptor:  the Descriptor should be set up with information that is _read_ by the StandardDescriptorProcessor and not vice versa.